deny.toml

```diff
@@ -96,7 +101,12 @@ highlight = "all"
 
 # List of crates to deny
 deny = [
-  { name = "openssl-sys" },
+  { crate = "openssl-sys" },
+  # Prevent adding new dependencies on protobuf that may use code with
+  # a security advisory in it (see `[advisories]`).
+  # If you *must* add a new crate to the "wrappers" here, carefully audit
+  # that it is *not* affected by any of the advisories above.
+  { crate = "protobuf:<3.7.2", wrappers = ["prometheus", "opentelemetry-prometheus"] },
```

Comment:
3.7.2 is not yet released, but the patch is going out in that version: https://github.com/stepancheg/rust-protobuf/pull/756